go/sgauth/credentials/file.go

```diff
@@ -23,6 +23,9 @@ import (
 // DefaultTokenURL is Google's OAuth 2.0 token URL to use with the JWT flow.
 const DefaultTokenURL = "https://accounts.google.com/o/oauth2/token"
```

Comment:
The default URL has changed. The new one is oauth2.googleapis.com/token. Please fix it.

Reply:
https://github.com/google/oauth2l/pull/50